unit/api/application_resource.py

```diff
@@ -22,8 +22,8 @@ def create(self, request: Union[CreateIndividualApplicationRequest, CreateBusine
         else:
             return UnitError.from_json_api(response.json())
 
-    def list(self, offset: int = 0, limit: int = 100) -> Union[UnitResponse[List[ApplicationDTO]], UnitError]:
-        response = super().get(self.resource, {"page[limit]": limit, "page[offset]": offset})
+    def list(self, offset: int = 0, limit: int = 100, email: Optional[str] = None, sort: Optional[str] = None) -> Union[UnitResponse[List[ApplicationDTO]], UnitError]:
+        response = super().get(self.resource, {"page[limit]": limit, "page[offset]": offset, "filter[email]": email, "sort": sort})
```

Comment:
This is the way we want to implement params. Please change it to look like this:
https://github.com/unit-finance/unit-python-sdk/pull/68/files